package.json

```diff
@@ -138,5 +139,6 @@
       "jest-watch-typeahead/filename",
       "jest-watch-typeahead/testname"
     ]
-  }
+  },
+  "sideEffects": false
```

Comment:
Perhaps we can move that line above the scripts so that it's not buried here.

Reply:
This way it’s alphabetical, and it’s usually last. Not sure what’s best

Reply:
Ah, didn't realize it was alphabetical. All good then!